src/Tasks/Microsoft.ManagedLanguages.targets

```diff
@@ -0,0 +1,64 @@
+﻿<!--
+***********************************************************************************************
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+This file defines common build logic for all managed languaged: C#, VisualBasic, F#
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project>
+   <!-- 
+        We are doing a cross-targeting build if there is a non-empty list of target frameworks specified
+        and there is no current target framework being built individually. In that case, a crosstargeting
+        project file like Microsoft.<language>.CrossTargeting.targets gets imported.
+   -->
+   <PropertyGroup Condition="'$(TargetFrameworks)' != '' and '$(TargetFramework)' == ''">
+      <IsCrossTargetingBuild>true</IsCrossTargetingBuild>
+   </PropertyGroup>
+
+   <PropertyGroup>
+      <InnerBuildProperty>TargetFramework</InnerBuildProperty>
+      <InnerBuildPropertyValues>TargetFrameworks</InnerBuildPropertyValues>
```

Comment:
Does structuring this as a property mean there can be only one dimension for inner builds? That's a current limitation of cross-targeting but people keep hoping we relax it somehow.

Reply:
I decided against generalization and went with the minimum to support the current sdk. The idea is we'll expand when necessary.
Ideally, `InnerBuildProperty` would be a set of global property names that define the inner build, and `InnerBuildPropertyValues` would contain all possible values somehow:
```
<InnerBuildProperties>TargetFramework;TargetPlatform ...
<InnerBuildPropertyValues>TargetFramework={net472;netcoreapp2.2}; TargetPlatform={win-x64;osx-x64} ...
```
Or, if one does not want a cross product of all values:
```
<InnerBuildProperties>TargetFramework;TargetPlatform ...
<InnerBuildPropertyValues>
  {
    {TargetFramework=net472; TargetPlatform=win-x64},
    {TargetFramework=netcoreapp2.2; TargetPlatform=osx-x64}
  }
```

But since these are not needed yet, not thinking about them speeds this current PR up. Let me know if we should think about them now though :)

Reply:
Another possible source of inspiration here is the Azure Pipelines YAML format for specifying a [variable matrix](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#matrix).